README.md

```diff
@@ -24,23 +24,27 @@ import { parquetWriteBuffer } from 'hyparquet-writer'
 
 const arrayBuffer = parquetWriteBuffer({
   columnData: [
-    { name: 'name', data: ['Alice', 'Bob', 'Charlie'], type: 'BYTE_ARRAY' },
+    { name: 'name', data: ['Alice', 'Bob', 'Charlie'], type: 'STRING' },
     { name: 'age', data: [25, 30, 35], type: 'INT32' },
   ],
 })
 ```
 
-Note: if `type` is not provided, the type will be guessed from the data. The supported parquet types are:
+Note: if `type` is not provided, the type will be guessed from the data. The supported types are a superset of the parquet types:
```

Comment:
maybe give more information about the mapping between these BasicType and the Parquet types (and also the JavaScript types?)

Reply:
maybe just referring to `basicTypeToSchemaElement`